app/controllers/api/v1/animation_datas_controller.rb

```diff
@@ -3,9 +3,11 @@ module V1
     class AnimationDatasController < ApplicationController
       skip_before_action :authenticate_user!
       def index
-        
-        @animation_record = UserAnimation.find_by("user_id = ? AND location = ? AND start_date <= ? 
-        AND end_date >= ?", params['user_id'], params['location'], Date.today, Date.today)
+
+        @animation_record = UserAnimation.
+        where("start_date <= :today_date AND end_date >= :today_date" , {today_date: Date.today}).
```

Comment:
remove hash syntax, write like
```
where("start_date <= :today_date AND end_date >= :today_date" , today_date: Date.today).
```
Also add 2 space indentation whenever you split line using .